### PR TITLE
From Day to Night - Creating Interactive Color Palettes with JavaScript IMC WebDev

### PR DESCRIPTION
**Pull request by emily-gui**
### Reference to issue
- #131 

### Description of the changes proposed in the pull request
- create proposal.md

### Reviewers requested:
- @minhtumn 
- @atuljayaram

### CODE REVIEW

File: IMC_WebDev/FromDayToNight-CreatingInteractiveColorPalettesWithJavaScript/proposal.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+**TITLE:**    
+From Day to Night - Creating Interactive Color Palettes with JavaScript
```
Comment (atuljayaram) [inline IMC_WebDev/FromDayToNight-CreatingInteractiveColorPalettesWithJavaScript/proposal.md:2]: Cool Title!

File: IMC_WebDev/FromDayToNight-CreatingInteractiveColorPalettesWithJavaScript/proposal.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+**TOPIC:**    
+JavaScript
+
+**DESCRIPTION (5-7+ sentences):**    
+This blog will be the last part of the tiny city blog series, which starts with setting up the HTML and CSS code in the blogs before. Here, the blog will provide a step-by-step tutorial to how to adding interactivity with JavaScript to the tiny city project. The viewer will learn how to adding a mouse-click feature to their project that will switch between a daytime and nighttime setting by changing the colors of their city. Since this blog will be catered towards people who are just getting into web development and coding, this blog will provide thorough explanations at each step and definitions for programming terms. By the end of this blog, the viewer will know the very basics of JavaScript, how to implement and incorporate a mouse-click interaction, and have completed their tiny city web development project.
```
Comment (atuljayaram) [inline IMC_WebDev/FromDayToNight-CreatingInteractiveColorPalettesWithJavaScript/proposal.md:8]: Seems like a fun idea!

**Comment by emily-gui** [comment]
Yay, thank you so much!

**Comment by emily-gui** [comment]
I finished my blog!
This is the loom video link: https://www.loom.com/share/0129881d019d4f4cb44075adea6e807a

**Pull request merged**